signingscript/test/test_task.py

```diff
@@ -1,25 +1,46 @@
-import pytest
+import unittest
```

Comment:
I would like these tests to remain pytest.  The syntax is cleaner, it's more modular, fixtures make setup and teardown much easier, etc. http://escapewindow.dreamwidth.org/247980.html#pytest